### PR TITLE
Explain what the wallet password does

### PR DESCRIPTION
**Pull request by Thanos107**
I have explained about the Wallet Password in this pull request.

I dealt with the open issue of "How wallet password works and does"

Explained how,

it is not the seed
it does not protect against a 5$ wrench
it might protect against a stranger walking up to an unlocked screen with a Bitcoin Core wallet running
it doesn't encrypt metadata or pubkeys
it results in loss of funds if the password is too complex and forgotten

Eager to hear from you.

### CODE REVIEW

File: doc/how-wallet-password-works.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+Certainly! A crypto wallet password serves as a critical layer of security for accessing and managing your cryptocurrency holdings stored within a wallet. Here's a comprehensive breakdown of its functions and how it operates:
```
Comment (luke-jr) [inline doc/how-wallet-password-works.md:1]: What?

File: doc/how-wallet-password-works.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+   - If a wallet is left open and unattended on an unlocked screen, someone nearby might access it without the password.
+   - Forgetting a complex password can result in the loss of funds, as there's often no way to recover it without the password or recovery seed.
+
+3. **Not Encryption for All Data:** The password typically encrypts the private keys and access to the funds but might not encrypt metadata or public keys associated with transactions. This means some information might still be accessible without the password.
+
+### Tips for Securing a Wallet Password:
+
+1. **Create a Strong Password:** Make sure your password is unique, complex, and not easily guessable. It should ideally be a combination of letters (upper and lower case), numbers, and special characters.
+
+2. **Backup and Recovery:** Always securely store and back up your wallet's recovery seed or mnemonic phrase. This allows you to recover your funds if you forget the password.
```
Comment (luke-jr) [inline doc/how-wallet-password-works.md:27]: No such thing

File: doc/how-wallet-password-works.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+1. **Purpose:** A wallet password primarily functions as a means of accessing and securing your wallet. It acts as a gatekeeper, preventing unauthorized access to your cryptocurrency funds stored within the wallet.
+
+2. **Authentication:** When you set up your wallet, you create a password. This password is required each time you want to access your wallet. It's used to authenticate and decrypt the wallet, providing access to your funds.
+
+3. **Limited Scope:** The password generally encrypts the private keys and provides access to your funds. However, it might not encrypt metadata or public keys associated with transactions. It protects against unauthorized access but has limitations in securing all wallet-related information.
+
+4. **Protection Level:** It guards against someone accessing your wallet software if it's open and unattended but may not protect against physical threats or someone with direct access to your unlocked device.
+
+### Seed Phrase (Recovery Phrase/Mnemonic Phrase):
```
Comment (luke-jr) [inline doc/how-wallet-password-works.md:45]: No such thing

**Review by luke-jr — CHANGES_REQUESTED** [review]

File: doc/how-wallet-password-works.md (hunk 1 above)
Comment (TheCharlatan) [inline doc/how-wallet-password-works.md:1]: This is a ChatGPT generated text.

Comment (Thanos107) [reply doc/how-wallet-password-works.md:1]: Actually, yeah. I used Chatgpt to refine it.

Should I not have done that?

Actually, this is my first pull request, so I don't have any idea how this works.

I will remove this pull request and push another one if that's okay with you.

**Comment by fanquake** [comment]
Closing in favour of #28974.

**Pull request closed without merging**